geo/src/algorithm/closest_point.rs

```diff
@@ -184,6 +184,7 @@ impl<F: GeoFloat> ClosestPoint<F> for Geometry<F> {
 
 #[cfg(test)]
 mod tests {
+    #![allow(clippy::excessive_precision)]
```

Comment:
Please leave a comment wherever you turn a warning off.

Reply:
@michaelkirk what kind of a comment and where would you like me to add? thx :)

Reply:
Can you clarify why it is OK to turn this warning off?

Reply:
ah, got it, thx.  Well... actually I do not know this for a fact TBH. This warning is about floating point precision, which is a very tricky subject (might even be CPU dependent?). My thinking is that the floats in the tests were copy-pasted from the failed assertions, so whatever got "printed" during the failed test was expected to be the "right" result. Technically I think we should switch to using [assert_approx_eq!()](https://crates.io/crates/assert_approx_eq), but that would be a separate PR.  I will leave a comment about that.

Reply:
If you remove the unnecessary precision, do the tests still pass?

Reply:
@frewsxcv @michaelkirk ok, I reduced precision in the floats and removed the `allow` attributes, the tests seemed to have passed ok.  Feel free to merge if OK